src/components/client-factory.ts

```diff
@@ -116,6 +119,34 @@ export class ClientFactory {
                 callback(err, response, body);
             });
         });
+
+        // matrix-js-sdk uses the `loglevel` logging library for it's logging
+        // but the only way to get it to log to winston is to modify the
+        // global methodFactory.
+        // To avoid potentially breaking other bridges, this is off by default.
```

Comment:
I don't understand where it's "off by default"

Reply:
err, I think I meant to delete this comment..